src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
+@import "utils/mixins";
+
+@import "./helpers/container";
+
+@import "./base/global";
+
+@import "./blocks/header";
+@import "./blocks/logo";
+@import "./blocks/navigation";
+@import "blocks/catalog";
```

Comment:
I think, you missed here `./` before `blocks/catalog`

Reply:
Thx 😊 Fixed ✅